tests/test_app_settings.py

```diff
@@ -17,7 +17,7 @@
     make_ingest_pipeline
 )
 
-TEST_STUDY_DIR = os.path.join(TEST_DATA_DIR, 'test_study')
+TEST_STUDY_DIR = os.path.join(TEST_DATA_DIR, 'simple_study')
```

Comment:
Is this all because `test_study` throws exceptions due to inconsistent data in merge results?

Reply:
yes